pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-plotting = [
+graphics = [
     # 2D chart plotting
```

Comment:
I would duplicate it for now and make sure that we encourage (in our docs) the usage of "graphics" rather than plotting. Eventually, completely remove it... but probably in the next big release, not right away